deploy/gcp/outputs.tf

```diff
@@ -1,32 +1,7 @@
-output "region" {
-  value = var.GCP_REGION
-}
-
 output "kubeconfig_file" {
   value = local.kubeconfig
 }
 
-output "tidb_version" {
-  value = var.tidb_version
-}
-
-output "monitor_ilb_ip" {
```

Comment:
Is it better that we follow the same rule https://github.com/pingcap/tidb-operator/blob/master/deploy/modules/share/tidb-cluster-release/outputs.tf#L15 here, so that users can still see the outputs if tidb-cluster deployed?

Reply:
how about abandoning it at all? otherwise, we need to maintain it in the future which is a burden for us.

Reply:
actually, I prefer deleting all unused terraform scripts/modules, then we can focus on infra provisioning.

Reply:
I agreed so. We could only provide the Kubernetes infra env by terraform and let user install operator and tidbcluster by themselves.

Reply:
OK, that will be a re-org of the terraform modules since we have to handle the asgs for operator and tidb-clusters.